web/src/App.js

```diff
@@ -28,6 +29,7 @@ function App() {
     <div>
       <Header />
       <Main />
+      <Comment />
```

Comment:
think this was also already mentioned but can add this to the professor page via the routing like in the professor_page branch instead of just having it render with the main page